Misc/Positions_Area Positions 4 External.txt

```diff
@@ -103,9 +95,6 @@ EISN_LN_CTR:Shannon Control:119.075:SNLN:A:EISN:CTR:-:-:6600:6677
 
 EISN_I_CTR:Shannon Information:127.500:SNI:I:EISN:CTR:-:-:6600:6677
 
-;Shannon - Withdrawn 2023/11
-;EISN_CTR:Shannon Control:131.150:SNE:E:EISN:CTR:-:-:6600:6677
-
 EISN_B_CTR:Shannon Control:134.275:SNB:B:EISN:CTR:-:-:6600:6667
```

Comment:
Same thing here, upto `Dublin`

Although there doesn't seem to be a `EISN_C_CTR` 8.33kHz definition, was that an omission or has it actually been removed @hazzas-99, seems you added it back but into the withdrawn section in your second PR #5063 when the changes were made.